api/src/pcapi/routes/backoffice_v3/forms/custom_reimbursement_rule.py

```diff
@@ -31,3 +38,55 @@ def is_empty(self) -> bool:
                 self.offerer.data,
             )
         )
+
+
+class EditCustomReimbursementRule(FlaskForm):
+    class Meta:
+        locales = ["fr_FR", "fr"]
+
+    offerer = fields.PCTomSelectField(
+        "Structure",
+        multiple=False,
+        choices=[],
+        validate_choice=False,
+        endpoint="backoffice_v3_web.autocomplete_offerers",
+    )
+    subcategories = fields.PCSelectMultipleField(
+        "Sous-catégories", choices=map(lambda s: (s.id, s.pro_label), subcategories_v2.ALL_SUBCATEGORIES)
+    )
+
+    rate = fields.PCDecimalField(
+        "Taux de remboursement %",
+        use_locale=True,
+        validators=[
+            wtforms.validators.InputRequired("Information obligatoire"),
+            wtforms.validators.NumberRange(min=0, message="Doit contenir un nombre positif"),
+            wtforms.validators.NumberRange(max=100, message="Doit contenir un nombre inférieur ou égal à 100"),
+        ],
+    )
+    start_date = fields.PCDateField("Date de début d'application")
+    end_date = fields.PCDateField(
+        "Date de fin d'application (optionnelle)", validators=(wtforms.validators.Optional(),)
+    )
+
+    def validate_start_date(self, start_date: fields.PCDateField) -> fields.PCDateField:
+        end_date_data = None
+        if self._fields["end_date"].data:
+            end_date_data = date_utils.get_day_start(self._fields["end_date"].data, finance_utils.ACCOUNTING_TIMEZONE)
+        start_date_data = date_utils.get_day_start(start_date.data, finance_utils.ACCOUNTING_TIMEZONE)
+
+        if end_date_data and (start_date_data > end_date_data):
+            raise wtforms.ValidationError("Ne peut pas être postérieure à la date de fin")
+
+        if start_date_data < date_utils.get_day_start(datetime.utcnow(), finance_utils.ACCOUNTING_TIMEZONE) + timedelta(
```

Comment:
Détail : je trouve personnellement ça plus clair en commençant par la diff :
```suggestion
        if start_date_data < date_utils.get_day_start(datetime.utcnow() + timedelta(days=1), finance_utils.ACCOUNTING_TIMEZONE):
```